external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceProvider.scala

```diff
@@ -538,6 +538,17 @@ private[kafka010] object KafkaSourceProvider extends Logging {
       .setIfUnset(ConsumerConfig.RECEIVE_BUFFER_CONFIG, 65536: java.lang.Integer)
       .build()
 
+  /**
+   * Returns a unique consumer group (group.id), allowing the user to set the prefix of
+   * the consumer group
+   */
+  private def streamingUniqueGroupId(parameters: Map[String, String],
```

Comment:
Sorry there isn't an automatic formatter for this... but use 4 space indentation for multi-line argument lists

https://github.com/databricks/scala-style-guide#spacing-and-indentation

Reply:
first arg should be on its own newline as well